_pages/community.md

```diff
@@ -41,6 +41,18 @@ FamilySearch has consolidated on one page the companies and websites committed t
 If you are associated with an application and want to add it to this listing or change its existing entry in this listing, send an email to <GEDCOM@FamilySearch.org>.
 
 
+## Steering Committee
```

Comment:
```suggestion
## GEDCOM Steering Committee
```
Suggest using the section title matching what we actually call the committee so a web search on the full name would find this section.